site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -242,6 +242,20 @@ populates chrome.runtime.lastError is not unchecked.
   </web-tab>
 </web-tabs>
 
+### Pass a message to a selected tab's content script {: #messaging }
+
+This example demonstrates how an extension's service worker can communicate with content scripts in specific browser tabs using `tabs.sendMessage`.
+
+```js
+function messagePasser(messageInfo) {
```

Comment:
Thoughts on calling this something like `sendMessageToActiveTab`? I think that's slightly more explicit about what it's going to do.